types/globals.d.ts

```diff
@@ -0,0 +1,18 @@
+interface topFilter {
+  name: string;
+  matches: number;
+  keywords: string[];
+}
+
+interface filterFace {
+  id: number;
+  name: string;
+  marketplace: string;
+  keywords: string[];
+  priceRange: string;
+  location: string;
+  active: boolean;
+  matchCount: number;
+}
+
+
```

Comment:
Types are upper-case. E.g. "TopFilter".

Also: It is not really recommended to just put types into the global file. I also did this in the skyblock project, but it is generally considered not nice.

Reply:
where would you prefer I put them

Reply:
Depends on the types. I would create a "types" folder, that contains files that export the types.

So in the types folder you would for example have a "FilterTypes" file that exports these types